spec/models/manageiq/providers/openshift/container_manager/refresh_parser_spec.rb

```diff
@@ -25,6 +25,7 @@
           :Author        => "ManageIQ team",
           :Size          => "123456",
           :DockerVersion => "1.12.1",
+          :Created       => Time.parse("2017-01-01").utc.to_s,
```

Comment:
This test currently looks like it checks you can round-trip Ruby's `Time#to_s` format.
This is an input to the test, right?  Can you change inputs to be literal strings matching the format seen in openshift API?
Expected `:registered_on` outputs can remain `Time` objects.